python/batch-predictor/Makefile

```diff
@@ -20,5 +24,13 @@ unit-test: type_check
 
 .PHONY: proto
 proto:
-	go get github.com/mitchellh/protoc-gen-go-json
-	protoc -I=. --python_out=merlinpyspark --mypy_out=merlinpyspark --go_out=pkg/ --go-json_out=pkg/ --go_opt=paths=source_relative spec/*.proto
+	@echo "Checking libprotoc version..."
+	@echo "protocPath=$(PROTOC_PATH)"
+	@PROTOC_VERSION=$$($(PROTOC_PATH) --version | grep -o '[0-9]\+\.[0-9]\+\(\.[0-9]\+\)\?'); \
+	if [ "$$PROTOC_VERSION" != "25.6" ]; then \
+		echo "Error: libprotoc version 25.6 is required. Found version $$PROTOC_VERSION."; \
```

Comment:
Wait, shouldn't these be `3.25.6` instead?

Reply:
no this is the correct version. I don't know why they omit the major version.

Reply:
Lol okay got it.